django/librarian/templates/librarian/components/edit_details.html

```diff
@@ -1,7 +1,7 @@
 {% load i18n %}
 
 <div id="librarian-details" {% if swap %}hx-swap-oob="true"{% endif %}>
-  {% include "components/messages.html" %}
+  {# {% include "components/messages.html" %} #}
```

Comment:
Commented this out because the messages also show up as toasts in the background.

TODO: Get the toasts to have a higher z-index (past the lightbox)